modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -270,7 +226,7 @@ export async function sendShareToBitgo(
  * @param recipientIndex - index of the recipient (1, 2, or 3)
  * @param recipientGpgPublicArmor - recipient's public gpg key in armor format
  * @param senderGpgPrivateArmor - sender's private gpg key in armor format
- * @returns { EncryptedNShare } encrypted Y Share
+ * @returns { EncryptedNShare } encrypted N Share
```

Comment:
NIT: may be we can avoid specifying type here since rest of the doc is not using it any ways, if we are specifying, then correct type should be `{ Promise<EncryptedNShare> }`